metaflow/plugins/argo/argo_workflows.py

```diff
@@ -977,11 +1007,23 @@ def _container_templates(self):
                                 kubernetes_sdk.V1VolumeMount(
                                     name="out", mount_path="/mnt/out"
                                 )
-                            ],
+                            ]
+                            + (
+                                [
+                                    kubernetes_sdk.V1VolumeMount(
+                                        name="argo-tmpfs-volume", mount_path=tmpfs_path
+                                    )
+                                ]
+                                if tmpfs_enabled
+                                else []
+                            ),
                         ).to_dict()
                     )
                 )
             )
+            if tmpfs_enabled:
+                template.empty_dir_volume("argo-tmpfs-volume", size_limit=tmpfs_size)
```

Comment:
what happens if the tmpfs_size is 0?

Reply:
if tmpfs_size = 0 implies no volume would be attached - we can move that logic in `def empty_dir_volume (self...` and avoid this if-else conditional here. - 

```
.empty_dir_volume("argo-tmpfs-volume", size_limit=tmpfs_size if tmpfs_enabled else 0)
```

Reply:
great point, this was a neat way to keep the method chaining tidy :)